server/lib/middleware.js

```diff
@@ -29,7 +29,7 @@ module.exports = function middlewareConstructor() {
      */
     checkForAuth: function(req, res, next) {
       if (!req.session.user) {
-        return res.redirect(301, "/");
+        return res.redirect(301, "/" + (req.localeInfo && req.localeInfo.lang || "en-US"));
```

Comment:
I think this would read better as a ternary:

`req.localeInfo ? req.localeInfo : "en-US"`

I've never seen this usage of `&&` such as this, and figured it'd output `"/true"`